### DISCUSSION TITLE
Why is there a warning under the "Show Dislike Count" option?

### DISCUSSION THREAD

**Discussion by mrbigmouth502** [polymorphicshade/NewPipe#373] [General]
It says: "WARNING: Your IP address will be visible to the API. Use at your own risk!"

Is this anything to be concerned about if I already use the Return YouTube Dislike addon for Firefox on desktop? I assume not, but then I wonder why the warning even exists in the first place. 

Asking here instead of in "Issues" because the option to ask a question just goes to the main NewPipe page, and my question isn't relevant to the main version of NewPipe.

**Comment by AnimeWaifuGirl** [comment]
It warns you if you enabled it tats you share your ip but it is normally thing
